src/crimson/net/ProtocolV2.cc

```diff
@@ -406,17 +406,13 @@ void ProtocolV2::dispatch_reset()
 
 void ProtocolV2::reset_session(bool full)
 {
+  server_cookie = 0;
```

Comment:
because we already have some related implementations with server_cookie, client_cookie and connect_seq, but they are stub code when implement lossy policy and cannot work properly...

Reply:
done. see https://github.com/ceph/ceph/pull/29378/commits/6a58ea5375979a99babee2217a0b2fa9a0c7d7e6